### PR TITLE
[TW-508] Uptime slack integration

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Added Slack integration setup to "Monitoring API Uptime" page
- Added steps for uptime monitor integration to "Slack" integration page
- Added linking uptime monitors to "Observing an API" page and updated screenshots

### CODE REVIEW

File: src/pages/docs/monitoring-your-api/uptime-monitors.md [hunk 1]
```diff
@@ -39,13 +40,13 @@ Create a new uptime monitor to track the availability of a single API endpoint,
 
 1. Select **Monitors** in the sidebar, and then select **+**.
 1. Select **Create an Uptime Monitor** in the right pane.
-1. Enter the **URL** you want to monitor. *Uptime monitors can only monitor URLs, API endpoints, and websites that are publicly available over the internet. Only HTTP and HTTPS are supported.*
+1. Enter the **URL** you want to monitor. *The URL, API endpoint, or website you want to monitor must be publicly available over the internet HTTP and HTTPS protocols are supported.*
```
Comment (HeatherStenson) [inline src/pages/docs/monitoring-your-api/uptime-monitors.md:43]:
```suggestion
1. Enter the **URL** you want to monitor. *The URL, API endpoint, or website you want to monitor must be publicly available over the internet. HTTP and HTTPS protocols are supported.*
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

File: src/pages/docs/designing-and-developing-your-api/observing-an-api.md [hunk 2]
```diff
@@ -56,48 +57,62 @@ You can choose between generating a collection from your API schema, using an ex
...
+1. On the API version **Monitoring** tab, next to **Collection-based Monitors**, select **Add Monitor** and choose **Add existing monitor**.
+1. Select a collection-based monitor from the list and select **Add Monitor**. (The list shows monitors available in your current workspace.)
 
-1. Select a monitor from the list and select **Add Monitor**. You will only see available monitors in your workspace in the list.
+For more about creating a monitor, see [Setting up a collection-based monitor](/docs/monitoring-your-api/setting-up-monitor/).
 
-For more about creating a monitor, see [Setting up a monitor](/docs/monitoring-your-api/setting-up-monitor/).
+## Linking uptime monitors
+
+You can link [uptime monitors](/docs/monitoring-your-api/uptime-monitors/) in your current workspace to an API version. Uptime monitors (open beta) continuously check the availability of a single API endpoint, website, or other URL. After linking an uptime monitor, you can view the current status and key statistics for the last 3 hours, such as availability, the number of downtime incidents, and average response time.
```
Comment (jkonrath-postman) [inline src/pages/docs/designing-and-developing-your-api/observing-an-api.md:76]:
```suggestion
You can link [uptime monitors](/docs/monitoring-your-api/uptime-monitors/) in your current workspace to an API version. Uptime monitors (open beta) continuously check the availability of a single API endpoint, website, or other URL. After linking an uptime monitor, you can view the current status and key statistics for the last three hours, such as availability, the number of downtime incidents, and average response time.
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM! A few minor corrections, but otherwise this is great!

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 3]
```diff
@@ -66,13 +66,13 @@ The Postman to Slack integration enables you to send Postman notifications to a
 
 ## Send your monitor run results to Slack
 
-> Before beginning, you must create a monitor. For more information, see [setting up a monitor](/docs/monitoring-your-api/setting-up-monitor/).
+> This integration works with collection-based monitors. Before you begin, make sure you've [created at least one collection-based monitor](/docs/monitoring-your-api/setting-up-monitor/). You can also set up a Slack integration for uptime monitors. Learn more about [integrating uptime monitors and Slack](/docs/monitoring-your-api/uptime-monitors/#getting-notified-about-downtime).
```
Comment (shashankawasthi88) [inline src/pages/docs/integrations/available-integrations/slack.md:69]: We need to add new section in the available integration for uptime monitor as well.

Comment (mark-anthony-postman) [reply src/pages/docs/integrations/available-integrations/slack.md]: Thanks Shashank! I've added the uptime integration steps to the Available Integrations > Slack page as well 🙂

**Pull request merged**